pkgs/applications/misc/citations/default.nix

```diff
@@ -0,0 +1,45 @@
+{ stdenv, lib, rustPlatform, fetchFromGitLab, meson, ninja, gettext, pkg-config
+, glib, gtk4, poppler, gtksourceview5, libadwaita, desktop-file-utils
+, wrapGAppsHook }:
+
+stdenv.mkDerivation rec {
+  pname = "citations";
+  version = "0.5.1";
+
+  src = fetchFromGitLab {
+    domain = "gitlab.gnome.org";
+    owner = "World";
+    repo = "citations";
+    rev = version;
+    hash = "sha256-QPK6Nw0tDdttUDFKMgThTYMTxGXsn5OReqf1LNAai7g=";
+  };
+
+  cargoDeps = rustPlatform.fetchCargoTarball {
+    inherit src;
+    name = "${pname}-${version}";
+    hash = "sha256-Kounxi4JxoU4+rWMWNB8rzTyG3MDKYD0OzYfAHwm6bY=";
+  };
+
+  nativeBuildInputs = [
+    gettext
+    glib
+    ninja
+    pkg-config
+    rustPlatform.cargoSetupHook
+    rustPlatform.rust.cargo
+    rustPlatform.rust.rustc
+    wrapGAppsHook
+    meson
+  ];
+
+  buildInputs =
+    [ desktop-file-utils glib gtk4 gtksourceview5 libadwaita poppler ];
```

Comment:
Tools used at build-time like `desktop-file-utils` should go to `nativeBuildInputs`, see https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/build-tools-in-build-inputs.md.

Also could you make the list one dependency per line? It will make diffs nicer to read and easier to check Git blame in the future.

Reply:
Nice Project.

Y this was the default doom emacs auto formatting of nix files.

Reply:
Do you have a formatter for this formatting? [Nixfmt](https://github.com/serokell/nixfmt) does not format it like that.